src/Lib/Server/Auth.hs

```diff
@@ -76,21 +77,21 @@ loginHandler LoginRequest{..} = timedAction "loginHandler" $ do
     $(logTM) DebugS $ ls $ "Incorrect password for user " <> loginRequestEmail
     throwError (notAllowed "Invalid Password")
   putSession userId Session { isLoggedIn = True }
-  token <- mkJWTToken (60 * 60 * 24) (JWTPayload userId)
-  return $ LoginResponse token
+  token <- mkJwtToken dayInSeconds (JwtPayload userId)
+  return $ LoginResponse (unJwtToken token)
```

Comment:
Could you also patch `LogingResponse` type to store `JwtToken` instead of `Text`?